phpcs.xml.dist

```diff
@@ -13,10 +17,8 @@
   </rule>
 
   <rule ref="Drupal.Files.TxtFileLineLength.TooLong">
-    <!-- Do not run this sniff on README.md as the file has long markup lines
-         which are allowed in the latest dev version of Coder.
-         @todo Update .travis.yml when Coder 8.x-2.13 is released and remove
-         this exclusion -->
+    <!-- Do not run this sniff on README.md as the file has long markup lines.
+     -->
```

Comment:
I know that the @ todo comment about updating .travis.yml is not correct as we do not need that anymore. But I do think it is worthwhile having a todo that states this rule does not need to be excluded when Coder 8.2.13 is released. We do still want to check that in general the README.md text lines do not exceed 80 chars, and we can allow this sniff after the next Coder release.

Reply:
That comment above does not show on the front Pull Request page, maybe I should have clicked 'Start a review'